release/release_test.go

```diff
@@ -186,6 +206,16 @@ func (s *ReleaseTestSuite) TestOnClassic(c *C) {
 	c.Assert(release.OnClassic, Equals, false)
 }
 
+func (s *ReleaseTestSuite) TestOnCoreDesktop(c *C) {
+	reset := release.MockOnCoreDesktop(true)
+	defer reset()
+	c.Assert(release.OnCoreDesktop, Equals, true)
```

Comment:
I don't fully understand the purpose of this test, I notice it was there before as well for classic.

Reply:
Yeah, it doesn't seem like a particularly useful test but it matches how the exisiting test is done for OnClassic.